openlmis-stock/src/main/java/org/smartregister/stock/openlmis/presenter/StockDetailsPresenter.java

```diff
@@ -192,8 +196,8 @@ private boolean processStock(String step, JSONObject jsonString, String provider
 
         List<LotDto> selectedLotDTos = LotFactory.gson.fromJson(lotsJSON, listType);
 
-        String tradeItem = extractValue(stepFields, TRADE_ITEM_ID);
-        String programId = extractValue(stepFields, PROGRAM_ID);
+        String tradeItem = extractValue(stepFields);
+        String programId = extractValue(stepFields);
```

Comment:
Return previous code that extracts tradeItem and programId

Reply:
Done!